integration/cri-o/cri-o.sh

```diff
@@ -68,9 +68,10 @@ IFS=''
 
 # Skip CRI-O tests that currently are not working
 pushd "${crio_repository_path}/test/"
+cp $GOPATH/src/${crio_repository}/test/ctr.bats $GOPATH/src/${crio_repository}/test/ctr_kata_integration_tests.bats
```

Comment:
minor comment (it's good as is):
crio_repository_path is $GOPATH/src/${crio_repository} and the script just pushd into it
so you can simply cp ctr.bats ctr_kata_integration_tests.bats (no need for prefix)

Reply:
Right. The "sed" line below was using the prefix, so I just copied it. But it's not needed.